packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -121,12 +121,12 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
 };
 
 /**
- * Props for dispalying a send button besides the text input area.
+ * Props for displaying a send button besides the text input area.
  *
- * @public
+ * @private
  */
 export type InputBoxButtonProps = {
-  onRenderIcon: (props: InputBoxButtonProps, isMouseOverSendIcon: boolean) => JSX.Element;
+  onRenderIcon: (isMouseOverIcon: boolean) => JSX.Element;
```

Comment:
should we say isHover?

Reply:
would be better, sure. Let me change that